src/validation.module.ts

```diff
@@ -1,8 +1,9 @@
-import { Module } from '@nestjs/common';
+import { Global, Module } from '@nestjs/common';
 import { GenericValidator } from './domain/schema/generic.validator';
 import { JsonSchemaService } from './domain/schema/json-schema.service';
 import { ValidationErrorFactory } from './domain/schema/validation-error-factory';
 
+@Global()
```

Comment:
I would create a folder for this class under `src` and for the `providers` that this class depends.

If this is a Global module I think it makes sense to not have a direct dependency to the `domain`

Reply:
I agree. I've created the folder `src/validation` and another folder `src/validation/providers/` with the contents of `src/domain/schema` 👍🏻 [2d26fea](https://github.com/5afe/safe-client-gateway-nest/pull/278/commits/2d26feaa7e29bf2e4f70bf72c0b82468211ebe3b)